frzr

```diff
@@ -23,8 +23,6 @@ frzr unlock (deployment) [Unlock the specified deployment, or the running one if
 frzr set-channel [Set the update channel stable/testing/unstable]
 frzr get-channel [Get the update channel currently in use]
 frzr version [Get the version of FRZR]
-frzr build-initramfs [Build the initramfs for the kernel]
-frzr configure-tweaks [Configure system specific quirks]
```

Comment:
How are we planning to enable/disable firmware overrides on the installer without this?

Reply:
Device-quirks PR has the functionality. It will be invoked on migrations inside a chroot. See here: https://github.com/ChimeraOS/chimeraos/blob/master/rootfs/usr/lib/frzr.d/install-0007-device-quirks-apply.migration